cmd/api-errors.go

```diff
@@ -2089,6 +2096,8 @@ func toAPIErrorCode(ctx context.Context, err error) (apiErr APIErrorCode) {
 		apiErr = ErrInvalidBucketName
 	case dns.ErrBucketConflict:
 		apiErr = ErrBucketAlreadyExists
+	case kms.KeyNotFoundException:
+		apiErr = ErrKMSKeyNotFoundException
```

Comment:
I still have this question on why we are introducing a new error that is wrapped on top of another error i.e kes. error we can directly use kes.ErrKeyNotFound here @poornas ?